vpc-endpoints.tf

```diff
@@ -45,18 +54,15 @@ resource "aws_vpc_endpoint_route_table_association" "public_s3" {
 data "aws_vpc_endpoint_service" "dynamodb" {
   count = var.create_vpc && var.enable_dynamodb_endpoint ? 1 : 0
 
-  service_type = var.dynamodb_endpoint_type
-  service      = "dynamodb"
+  service = "dynamodb"
```

Comment:
I think we can add `service-type` filter as you do for S3 VPC endpoint to be forward-compatible with the upcoming DynamoDB in case it happens anytime (users won't have to do anything to get it to work and it won't break their setup).

Reply:
sounds good - just pushed up the update for this